### PR TITLE
refactor(analytics): Chrome -> Cordova analytics

### LINKED ISSUE #2: Replace chrome-platform-analytics
Metrics to port:

* [x] Exceptions ([exceptionhandlerDecorator](https://github.com/eHealthAfrica/move/blob/d79cf62078058286b3967507204c9c10fc078485/app/scripts/decorators/exceptionhandler-decorator.js))
* [x] Click events ([gaClick](https://github.com/eHealthAfrica/move/blob/d79cf62078058286b3967507204c9c10fc078485/app/scripts/directives/analytics.js))
* [x] Page views ([postAppView](https://github.com/eHealthAfrica/move/blob/d79cf62078058286b3967507204c9c10fc078485/app/scripts/services/tracking-service.js#L100-107))

Test:

* [ ] Offline caching (http://stackoverflow.com/questions/6618719/how-does-google-analytics-for-android-handle-offline-app-usage)

Set up:

* [x] New app properties in GA

### PR DESCRIPTION
**Pull request by tlvince**
Uses https://github.com/danwilson/google-analytics-plugin, which in turn uses
Android's/Google Play Service's native Google Analytics service, which means we
get offline handling for free, at slight cost of a bigger APK (it pulls in the
Google Analytics SDK jar), plus a dependency on Google Play Service for
analytics reporting (which should be moot as we control the devices the app is
installed on and generally, Play Services is only missing on modded devices).

Closes #2.

### CODE REVIEW

**Comment by tlvince** [comment]
@karlwestin I'm going to give this some more testing on another device of mine (later tonight) and wait until the events are consolidated in the Google Analytics dash (4-6 hours?), but could you give it a once over?

File: app/scripts/services/tracking-service.js [hunk 1]
```diff
@@ -1,139 +1,47 @@
...
-    if (utility.has($window, 'analytics')) {
-      var service = $window.analytics.getService(config.analytics.service);
-      tracker = service.getTracker(config.analytics.propertyID);
-      registerListeners();
+    this.trackEvent = angular.noop;
+    this.trackException = angular.noop;
 
-      $rootScope.$on('MEMORY_STORAGE_LOADED', setUUID);
+    if ($window.analytics) {
+      $window.analytics.startTrackerWithId(config.analytics.propertyID);
```
Comment (karlwestin) [inline app/scripts/services/tracking-service.js]: this needs to go on deviceready.. or does the app wait with bootstrapping til deviceready? Just wasnt sure

Comment (tlvince) [inline app/scripts/services/tracking-service.js]: Yeah, I'll do an explicit `angular.bootstrap` on `deviceready` (or do we it handle it differently in sense/sense-id)?

Currently, we just assume Cordova has loaded before Angular, which in practice, it seems to be more often than not, but better to be explicit…

**Comment by karlwestin** [comment]
w/ comment about this looks good to me! Nice!

File: app/scripts/services/tracking-service.js (hunk 1 above)
Comment (tlvince) [inline app/scripts/services/tracking-service.js]: https://github.com/eHealthAfrica/move/issues/41

**Pull request merged**